src/Features/CSharp/Portable/UseSimpleUsingStatement/UseSimpleUsingStatementCodeFixProvider.cs

```diff
@@ -84,9 +84,23 @@ private static IEnumerable<StatementSyntax> Expand(UsingStatementSyntax usingSta
             var result = new List<StatementSyntax>();
             Expand(result, usingStatement);
 
+            var blockStatement = usingStatement.ChildNodes().Where(x => x.Kind() == SyntaxKind.Block);
```

Comment:
so this won't work for a lot of scenarios.  for example:

```c#
using (var x = y)
using (var a = b)
{
     // whatever
}
```

You'll probably need to update `private static void Expand(List<StatementSyntax> result, UsingStatementSyntax usingStatement)` instead.

Reply:
what you want to do in that function is collect the trivia to preserve when handling the block-syntax case.  it will need to return that trivia here.  that trivia will then need to be returned higher up the stack up to:

```c#
                if (statementToUpdate is UsingStatementSyntax usingStatement &&
                    usingStatement.Declaration != null)
                {
                    var updatedStatements = currentBlock.Statements.ReplaceRange(
                        statementToUpdate,
                        Expand(usingStatement));
                    return currentBlock.WithStatements(updatedStatements);
                }
```

At that point, it will need to take the trivia and assign it to the next statement in the block